router/router.go

```diff
@@ -179,3 +202,13 @@ func (r *router) handleServiceRequest(w http.ResponseWriter, req *http.Request)
 	defer cancel()
 	r.gateway.HandleServiceRequest(reqCtx, req, w)
 }
+
+// handleSanctionedEndpoints returns a JSON list of sanctioned endpoints
+// This will eventually be removed in favour of a metrics-based approach.
```

Comment:
If it'll be eventually removed, it's a TODO.

IMO, we won't.  

It's a debug tool that's easier to use. Doesn't replace metrics.